spec/features/course/admin/codaveri_settings_spec.rb

```diff
@@ -27,7 +27,7 @@
         expect_toastify('Your changes have been saved.')
 
         expect(option).not_to be_checked
-        expect(course.reload.codaveri_itsp_enabled?).to eq(nil)
+        expect(course.reload.codaveri_itsp_enabled?).to eq(false)
```

Comment:
Nit: `.to be_falsey`?

Reply:
As discussed, we expect the result to be `false` only